packages/import-utils/src/api/define.ts

```diff
@@ -14,6 +15,9 @@ function compress(input: string) {
     .replace(/=+$/, ``); // Remove ending '='
 }
 
-export function getParameters(parameters: { files: IFiles }) {
+export function getParameters(parameters: {
+  files: IFiles;
+  template: ITemplate;
```

Comment:
I think the template is optional, other than that, good to go!

```suggestion
  template?: ITemplate;
```